cmd/sns-sqs/main.go

```diff
@@ -1,98 +0,0 @@
-package main
```

Comment:
@roblaszczak I found these examples useful. Maybe we can put them in an example folder....

Reply:
yeah, we'll put a separate examples here as for other pubsubs: https://github.com/ThreeDotsLabs/watermill/tree/master/_examples/pubsubs